pkg/utils/k8sutil.go

```diff
@@ -453,21 +453,33 @@ func CreateK8sCustomResource(runtime, handler, file, funcName, funcType, topic,
 		Do().Into(&f)
 	if err != nil {
 		if k8sErrors.IsNotFound(err) {
+			funcLabels := map[string]string{}
+			for _, label := range labels {
+				k, v := getKV(label)
+				funcLabels[k] = v
+			}
+
+			funcAnnotation := map[string]string{
+				"Description": description,
```

Comment:
Oops sorry I missed this comment. This is from Seb suggestion:

> Description
An annotation to the function object
kubeless function create --description "this is whatever" .... --from-file... foobar

But yes I agree with you. This should be a field in function spec rather than an annotation. Fix it now. Thanks Gus 👍